# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 3b1dbe909aaf
# Full Hash: 3b1dbe909aaf1982c72db1e5bc8cb35208ae6c77
# Author: Dimi Lee <dlee@mozilla.com>
# Date: 2020-04-23 21:43:09
# Regressor Bug: 1616775
# File Overlap Count: 1
# Description:
#   Bug 1616775 - P4. Add CompleteAllowAccessFor IPDL r=timhuang,baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71013
# ==============================================================================

diff -r 123b6c3a0ede -r 3b1dbe909aaf dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Thu Apr 23 14:28:41 2020 +0000
+++ b/dom/ipc/ContentParent.h	Thu Apr 23 14:25:16 2020 +0000
@@ -1239,6 +1239,13 @@
       const nsCString& aTrackingOrigin, const int& aAllowMode,
       FirstPartyStorageAccessGrantedForOriginResolver&& aResolver);
 
+  mozilla::ipc::IPCResult RecvCompleteAllowAccessFor(
+      const MaybeDiscarded<BrowsingContext>& aParentContext,
+      uint64_t aTopLevelWindowId, const Principal& aTrackingPrincipal,
+      const nsCString& aTrackingOrigin, uint32_t aCookieBehavior,
+      const ContentBlockingNotifier::StorageAccessGrantedReason& aReason,
+      CompleteAllowAccessForResolver&& aResolver);
+
   mozilla::ipc::IPCResult RecvStoreUserInteractionAsPermission(
       const Principal& aPrincipal);
 